libraries/testing/tester.cpp

```diff
@@ -86,7 +99,14 @@ namespace eosio { namespace testing {
       return fc::json::from_string( read_gzipped_snapshot(fn) );
    }
 
-   const fc::microseconds base_tester::abi_serializer_max_time{1000*1000}; // 1s for slow test machines
+   void write_binary_snapshot( const char* fn, const std::string& snapshot ) {
+      return write_gzipped_snapshot( fn, snapshot );
+   }
+
+   void write_json_snapshot( const char* fn, const fc::variant& snapshot) {
+      return write_gzipped_snapshot( fn, fc::json::to_string(snapshot, fc::time_point{fc::microseconds{1000*1000}}) );
```

Comment:
Use `abi_serializer_max_time`

Reply:
I'm just going to remove the write snapshot functions introduced in this PR because they don't seem to serve any purpose currently and in fact aren't even used.